transforms/integration/transforms/set-value.js

```diff
@@ -67,6 +68,16 @@ function hasFluentTriggerCall(j, path) {
     ;
 }
 
+function hasFluentTriggerAndMoreCall(j, path) {
+  const greatGrandParent = path.parentPath.parentPath.parentPath.node;
+  return greatGrandParent
+    && j.MemberExpression.check(greatGrandParent)
+    // add additional method names to the following
+    // to exempt them from processing e.g. focus()
+    && /^focusout|blur$/i.test(greatGrandParent.property.name)
```

Comment:
Do we actually have to check this? For one thing there could be other cases of additonal fluent calls besides `focusout` and `blur`. 

I am not sure exactly at which line of code where you were hitting the exception reported in #36, but my understanding was that any additional fluent call after `.trigger()` will lead to this, doesn't it? So would just checking `j.MemberExpression.check()`  be enough here, and cover more invalid cases?

I might miss something though, not fully understanding the problem eventually...

Reply:
We probably don't need to check for this. I wanted to be cautious and whitelist the keywords. I'll remove it.
The line I hit the exception was the 3 call chain [here](https://github.com/simonihmig/ember-test-helpers-codemod/pull/37/files#diff-e154e411c6db60dfd89af0762c42ed6dR19).